tests/acceptance/bootstrap/GraphContext.php

```diff
@@ -2891,6 +2893,50 @@ public function userListsTheActivitiesOfSpaceUsingTheGraphApi(string $user, stri
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @When the public tries to check the activities of space :spaceName owned by user :user with password :password
+	 *
+	 * @param string $spaceName
+	 * @param string $user
+	 * @param string $password
+	 *
+	 * @return void
+	 */
+	public function thePublicTriesToCheckTheActivitiesOfSpaceOwnedByUserWithPassword(string $spaceName, string $user, string $password): void {
+		$spaceId = $this->spacesContext->getSpaceIdByName($user, $spaceName);
+		$password = $this->featureContext->getActualPassword($password);
+		$response = GraphHelper::getActivities(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			"public",
+			$password,
+			$spaceId
+		);
```

Comment:
not sure but we can do this```suggestion
		$response = GraphHelper::getActivities(
			$this->featureContext->getBaseUrl(),
			$this->featureContext->getStepLineRef(),
			"public",
			$this->featureContext->getActualPassword($password),
			$this->spacesContext->getSpaceIdByName($user, $spaceName)
		);
```

Reply:
Yeah sure we can do it. Thanks for the suggestion!